core/src/toga/command.py

```diff
@@ -114,17 +114,17 @@ def __hash__(self) -> int:
         return hash(self.key)
 
     def __lt__(self, other: object) -> bool:
-        if not isinstance(other, (Group, Command)):
+        if not isinstance(other, Group | Command):
```

Comment:
Well, this is a new one to me. But I think the tuple syntax is still marginally easier to read, because the open parenthesis tells you what it's going to be before you've even read `Group`.

Reply:
OK fair enough, I'll accept it as it is.